src/App.js

```diff
@@ -1,14 +1,23 @@
 import React, { Component } from 'react';
 import { data } from './data';
-import { ShoppingList } from './components';
+import { Wishlist, Cart } from './components';
 class App extends Component {
-  state = { list: data };
+  state = { list: data, shoppingCart: [] };
+
+  addToList = (id, status) => _ => {
+    const { list } = this.state;
+    if (list[id].id === id) list[id].inCart = status;
```

Comment:
Mutating state directly is a big no. 
```
const { list } = this.state;
const newList = list.map(item => item.id === id ? { ...item, inCart: !item.inCart } : item)
this.setState({ list: newList })
```

Reply:
You also won't need that second argument with this method. The fewer arguments that a function relies on, the better.